### DISCUSSION TITLE
Sending emails via the REST API

### DISCUSSION THREAD

**Discussion by Paultje52** [modoboa/modoboa#2277] [Help] [answered]
Hello. I was wondering if it was possible to send/recieve messages via the rest API. I want to build my own basic mail client and use modoboa as the "backend", so I don't have to worry about protocols, like SMTP. Is this possible, and if yes, how?

Edit: When looking in the network tab in dev-tools I can see the sending emails part, but I don't want to reverse engineer the whole client just to make my own simple mail client

**Comment by H-A-L-I-S** [comment, marked as answer]
You can use SMTP instead of the API. Why using a API if you have a standard protocol for that?
